src/Extensions.Hosting/Extensions.Hosting.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
```

Comment:
I MUCH prefer `Nullable` and we have (very slowly) started converting individual files to it and new testing project have enabled it. I do like implicit usings as well since we can save some generally skipped over space and get into the code quicker but I'm less attached to it.